docs/pages/User/GettingStarted/peer.md

```diff
@@ -3,51 +3,180 @@ title: Peer to a foreign cluster
+
+First, you have to retrieve the Cluster ID (i.e. a unique identifier generated by Liqo for identification purposes during the peering process) of your cluster:
 ```
-kubectl get foreignclusters ff5aa14a-dd6e-4fd2-80fe-eaecb827cf55 --template={{.spec.join}}
-true
+kubectl get configmap -n liqo cluster-id -o jsonpath='{.data.cluster-id}'
 ```
 
+Now, it is possible to configure the records necessary to enable the DNS discovery process.
+In the following, we present a `bind9`-like configuration for an hypothetical domain `example.com`. It exposes one Liqo-enabled cluster named `liqo-cluster`, with the API server accessible at `1.2.3.4:6443`.
+Remember to adapt the configuration according to your setup.
+```txt
+example.com.                           PTR     liqo-cluster._liqo._tcp.example.com.
+
+liqo-cluster._liqo._tcp.example.com.   SRV     0 0 6443 liqo-cluster-api.server.example.com.
+
+liqo-cluster._liqo._tcp.example.com.   TXT     "id=${YOUR_CLUSTER_ID}"
+                                               "namespace=liqo"
+
+liqo-cluster-api-server.example.com.   A       1.2.3.4
+```
+
+{{%expand "Expand here to know more about the meaning of each record." %}}
+
+* the `PTR` record lists the Liqo clusters exposed for the specific domain (e.g. `liqo-cluster`, reachable in `TCP`).
+* the `SRV` record specifies the network parameters needed to connect to the Kubernetes API server of the cluster.
+  Specifically, it has the following format:
+  ```txt
+   <cluster-name>._liqo._tcp.<domain>. SRV <priority> <weight> <api-server-port> <api-server-name>.
+  ```
+  where the priority and weight fields are unused and should be set to zero. In this case, the API server is reachable at the address `liqo-cluster-api.server.example.com` through port `6443`.
+* The `A` record assigns an IP address to the DNS name of the Kubernetes API server, in this case `1.2.3.4`.
+  Although, for the sake of simplicity, this example configures an `A` record, an indirect `CNAME` record would also work.
+* the `TXT` record is opaque to the DNS system and it is used to store the `Cluster ID` and the `Liqo Namespace` (i.e. where the Liqo components are installed) parameters associated with the cluster.
+
 {{% /expand %}}
-<!-- TODO: The above sentence looks not obvious for an occasional user. Please be more user-friendly. -->
 
+### Connect to a remote cluster
+
+In order to leverage the DNS discovery to peer to a remote cluster, it is necessary to specify the remote domain.
+This operation can be easily performed through the graphical dashboard: click on the "+" icon located near Available Peers and then select "Add domain".
+Here, you need to configure the following parameters:
+1. **Domain**: the domain where the cluster you want to peer with is located.
+2. **Name**: a mnemonic name to identify the domain.
+3. **Join**: to specify whether to automatically trigger the peering procedure.
+
+{{%expand "Using kubectl, it is also possible to perform the same configuration." %}}
```

Comment:
This looks very good to me: add also kubectl commands in this guide, but hiding the text so that it looks available only to most experienced people.